### ISSUE TITLE
pip install ends up in error

### ISSUE THREAD

**Issue by fazpu** [gopaycommunity/gopay-python-api#2]
the python 3.5 pip install on Ubuntu 16.04 throws:

```
Collecting gopay==1.1.0 (from -r /home/merchang/code/requirements/base.txt (line 22))
  Downloading gopay-1.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-qh0r3wdc/gopay/setup.py", line 2, in <module>
        from pypandoc import convert
    ImportError: No module named 'pypandoc'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-qh0r3wdc/gopay/

```

**Comment by dorianpodulka** [comment]
Hello,
any updates on this? I want to use Gopay on Heroku, but I can't make it to install Pypandoc :( Is this requirement necessary?
Or did you find any other solutions?
Thanks in advance.

**Comment by HonzsSedlomn** [comment]
@dorianpodulka Just ask them to install it for you.

**Comment by xPapp** [comment]
pip install gopay

Collecting gopay
  Using cached gopay-1.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-s1dfdy1r/gopay/setup.py", line 2, in <module>
        from pypandoc import convert
    ImportError: No module named 'pypandoc'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-s1dfdy1r/gopay/

any updates?

**Comment by JurajKavka** [comment]
... as a solution install `pandoc` (with linux package manager) and `pypandoc` (with pip) to system.

we had also issues with that `pypandoc` reuirement. Only for one row in `setup.py`, the `pandoc` package with all the `haskell-something` dependencies needs to be installed in system. In case of docker container, it is about 200MB of unneeded stuff.

Please, if it is possible, remove that dependency.  Thank you. :)

**Comment by FSichinger** [comment]
https://github.com/gopaycommunity/gopay-python-api/commit/f370e1cd4fcbadc333e7c24a55966c3c665d6bc8

The dependency has been removed
a new version released 1.2.3
